src/svg/ds6/icons.svg

```diff
@@ -404,4 +404,12 @@
     <symbol id="icon-user-profile" viewBox="0 0 22 22">
         <path stroke="#000" stroke-width=".2" d="M11 19.925a8.925 8.925 0 100-17.85 8.925 8.925 0 000 17.85zm0 1.575C5.201 21.5.5 16.799.5 11S5.201.5 11 .5 21.5 5.201 21.5 11 16.799 21.5 11 21.5zm6.052-3.829a2.21 2.21 0 00-1.317-2.025l-2.439-1.023a1.816 1.816 0 01-1.116-1.727 1.81 1.81 0 01.708-1.406c.961-.735 1.365-1.94 1.055-3.146-.326-1.264-1.48-2.147-2.807-2.147-.81 0-1.57.317-2.141.893a2.934 2.934 0 00.306 4.4c.437.335.695.848.708 1.406a1.816 1.816 0 01-1.116 1.727l-2.439 1.023a2.2 2.2 0 00-1.313 1.892.744.744 0 11-1.484-.09 3.688 3.688 0 012.21-3.177l2.453-1.03a.327.327 0 00.202-.311.322.322 0 00-.122-.248 4.423 4.423 0 01-1.744-3.529 4.394 4.394 0 011.286-3.12A4.467 4.467 0 0111.135 4.7a4.373 4.373 0 014.247 3.269c.457 1.773-.169 3.622-1.593 4.713a.322.322 0 00-.122.248.327.327 0 00.202.312l2.453 1.029a3.706 3.706 0 012.217 3.4h-1.487z"/>
     </symbol>
+<!--alias of chevron-left-small-->
+    <symbol id="icon-carousel-prev" viewBox="0 0 8 14">
+        <path fill-rule="evenodd" d="M7 13.94a1 1 0 01-.71-.29l-6-6a1 1 0 010-1.41l6-6A1 1 0 017.7 1.65L2.41 6.94l5.29 5.29a1 1 0 01-.7 1.71z"/>
```

Comment:
@agliga Just to clarify, do you mean replacing `path` with `use`?

Reply:
@agliga Changing to use seems to cause issues when running the build images script. It unexpectedly updated the base64 and and the background icon for ds6 stopped working. I'll leave as is for now but create a ticket for us to address this later.

![Screen Shot 2019-11-20 at 11 58 02 AM](https://user-images.githubusercontent.com/38065/69273444-3c688b00-0b8d-11ea-92e2-7a0e9221f922.png)

![Screen Shot 2019-11-20 at 11 58 36 AM](https://user-images.githubusercontent.com/38065/69273452-3f637b80-0b8d-11ea-994e-843dc61f5894.png)